backend/src/main/java/com/javahelp/backend/TokenAuthManager.java

```diff
@@ -0,0 +1,33 @@
+package com.javahelp.backend;
```

Comment:
As a use case, this should probably go somewhere like `com.javahelp.backend.domain.user`, etc...